### ISSUE TITLE
Npm start fails every time.

### ISSUE THREAD

**Issue by RancorSnp** [HitomaruKonpaku/verniy-bot#33]
Hi, 

So I have to my knowledge followed the readme step by step exactly as written, 

git clone, cd to the location, npm install. Set up config.yaml and .env invited my version of the bot to the server, 

> verniy-bot@1.10.2 build
> tsc

npm run deploy-commands

says it sucessfully registered guild commands and application commands, but when I try to start 

every single time I get : 

[Nest] 34376  - 20.02.2024, 21:40:55   ERROR [TypeOrmModule] Unable to connect to the database. Retrying (9)...
QueryFailedError: SQLITE_ERROR: index IDX_142d9c520756c478f9bbeb9d22 already exists
    at handler (C:\Users\x\verniy-bot\node_modules\typeorm\driver\sqlite\SqliteQueryRunner.js:81:26)
    at replacement (C:\Users\x\verniy-bot\node_modules\sqlite3\lib\trace.js:25:27)
    at Statement.errBack (C:\Users\x\verniy-bot\node_modules\sqlite3\lib\sqlite3.js:15:21)
[Nest] 34376  - 20.02.2024, 21:40:55   ERROR [ExceptionHandler] SQLITE_ERROR: index IDX_142d9c520756c478f9bbeb9d22 already exists
QueryFailedError: SQLITE_ERROR: index IDX_142d9c520756c478f9bbeb9d22 already exists
    at handler (C:\Users\x\verniy-bot\node_modules\typeorm\driver\sqlite\SqliteQueryRunner.js:81:26)
    at replacement (C:\Users\x\verniy-bot\node_modules\sqlite3\lib\trace.js:25:27)
    at Statement.errBack (C:\Users\x\verniy-bot\node_modules\sqlite3\lib\sqlite3.js:15:21)
    
    It's especially peculiar since it says it is trying to create an index that already exists, but the database it creates is completely empty, 0b. I've been trying to troubleshoot it for 2 days now, but I simply can't figure out what is going wrong. 
    
    To my knowledge I followed every step of the readme, exactly as written

**Comment by HitomaruKonpaku** [comment]
@RancorSnp hmm i'm not sure about this one 🤔 

If this is a completely new bot, try to delete the file `verniy-bot/db/database.sqlite` and try again

**Comment by RancorSnp** [comment]
I've tried deleting the database file with no effect. I tried to remove the bot from the server, remove all of the bot files from my PC, and install the bot from the start. Same thing. I tried to do it through the docker, but each and every time I am getting the same error.

Is there any way I can customize the message the bot sends when it reports that a Twitcast stream is live without being recognized by the bot as the owner? As clearly bot has no intention of starting from my PC, and using the existing version of the bot seems to only allow the owner to access certain commands. 

I mostly just need it to mention the role name when detecting a stream
